src/hypermedia-constants.js

```diff
@@ -68,6 +68,7 @@ export const Rels = {
 	// Assignments
 	Assignments: {
 		instructions: 'https://assignments.api.brightspace.com/rels/instructions',
+		releaseConditions: 'https://assignments.api.brightspace.com/rels/release-conditions',
```

Comment:
I'm thinking that since this is a preliminary crappy version of it, I'm going to pick a different rel so as to not steal the good one.